docs/terminology/dmsrules.md

```diff
@@ -0,0 +1,58 @@
+# DMS Rules Details
+
+## View
+
+### Filter
+Filters are used to specify which nodes or edges should be returned when querying a view. This is difficult to
+set manually, thus, unless you know what you are doing, we recommend using the default set by **NEAT**. You select
+the default by leaving the filter empty.
+
+#### What is supported?
+While **NEAT** DMSRules are one-to-one with the CDF API specification for creating Data Models, the view filter is the
+exception. If you compare to the [API specification for views](https://api-docs.cognite.com/20230101/tag/Views/operation/ApplyViews)
+you will notice that `filter` is a very flexible parameter were you can specify an arbitrary complex filter.
+**NEAT** has an opinionated approach to creating the view filter.
+
+**NEAT** supports two filters
+
+* `hasData` in `containers` - This filter returns all nodes/edges that have data in the specified containers.
+* `NodeType` filter - This filter returns all nodes of a specific type. **NEAT** supports specifying multiple node types.
+
+#### Default Filter (Smart Defaults)
+
+The default filter set by **NEAT** is set based on the data model type, whether the view is mapping to containers or not, and
+whether the view is mapping to containers in another data model.
+
+* If the data model type is `solution` and the view has properties mapping to containers in another data model,
+  the default filter is `nodeType` filter with all node ids matching the id of the container in the other data model.
+* If the data model type is `solution` and the view has properties mapping only to containers in the same data model,
+  the default filter is `hasData` filter with all containers in the data model.
```

Comment:
this typically occurs when solution is partially based on enterprise data model, and partially of solution specific extensions of enterprise data model (thus contain additional containers beyond enterprise model containers)